code/game/objects/structures/watercloset.dm

```diff
@@ -414,6 +414,16 @@
 		user << "<span class='warning'>Someone's already washing here!</span>"
 		return
 
+	if(istype(O, /obj/item/weapon/reagent_containers/food/snacks/monkeycube))
+		if (/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped)

```

Comment:
Moreso when we want to check if ```O``` is wrapped (and not  checking if wrapped is a variable of monkeycude or whatever checking a typepath do).